indexFormat/implementation.md

```diff
@@ -0,0 +1,78 @@
+# Building an LSIF exporter
+
+By building an LSIF (Language Server Index Format) exporter for your language server, users can use [Rich Code Navigation](https://code.visualstudio.com/blogs/2018/12/04/rich-navigation) on their pull requests.
+
+If you are new to LSIF, [the specification page](specification.md) covers the motivation and implementation details of the protocol.
+
+In this guide, we cover how an LSIF implementation can be used in the context of Rich Code Navigation.
+
+## Rich Code Navigation scenario
+
+With Rich Code Navigation, users can use navigation features on their pull requests, without requiring a local checkout. In this scenario, the navigation features are powered through an external service. This service uses a pre-built LSIF index. The index can be generated at a variety of places, like for instance the CI pipeline of the repo:
+
+1. User creates a pull request
+1. The CI tool builds this pull request
+1. The LSIF exporter runs in the CI pipeline and generates the LSIF file
+
+## LSIF exporter implementations
+
+| Language | Repository |
+|--|--|
+| TypeScript/JavaScript | [lsif-node](https://github.com/Microsoft/lsif-node) |
+
+> Are we missing an implementation? Create a new issue on GitHub.
+
+## LSIF implementation skeleton
+
+As detailed in the specification, the LSIF exporter consists of two tools: the index exporter and the package manager linker.
+
+### Index exporter
```

Comment:
Could you make a section called terms or something where you have definitions of the terminology we use like Index Exporter and Package Manager Linker. I felt a little confused when reading the description and then realized it's meant to be a definition of the title since everything that came before was a topic header.